apps/api/src/services/data-provider/manual/manual.service.ts

```diff
@@ -166,12 +166,16 @@ export class ManualService implements DataProviderInterface {
         }
       });
 
-      for (const { currency, symbol } of symbolProfiles) {
+      for (const { currency, symbol, scraperConfiguration } of symbolProfiles) {
         let marketPrice =
           marketData.find((marketDataItem) => {
             return marketDataItem.symbol === symbol;
           })?.marketPrice ?? 0;
 
+        if (scraperConfiguration.mode === 'instant') {
```

Comment:
I think you need to call `await Promise.all()` outside of the loop, otherwise it needs to wait for each scraping requests to finish.

You could use `filter()` to find the symbolProfiles with a `scraperConfiguration`:
```
symbolProfiles.filter(({scraperConfiguration}) => { return !!scraperConfiguration?.url })
```